src/backend/language.cpp

```diff
@@ -11,23 +11,44 @@
 **********/
 
 #include "language.h"
+#include <iostream>
 
 #include <QHash>
 #include <QLocale>
 #include <QObject>
 #include <QtGlobal>
+#include <QIODevice>
+#include <QStringLiteral>
+#include <QDir>
+#include <QFile>
+#include <QJsonDocument>
+#include <QJsonObject>
+#include <QJsonArray>
+#include <QJsonValue>
+#include <QDebug>
 #include <utility>
 #include "../util/btassert.h"
 #include "managers/btlocalemgr.h"
 
 // Sword includes:
 #include <swlocale.h>
 
+// iso-codes includes
+#include <libintl.h>
+
```

Comment:
I suggest ordering this with the rest of the includes in alphabetical order. We only keep Sword includes separately, because of Sword code quality issues (i.e. we often need to wrap these between `#pragma gcc diagnostic` hacks to silence compiler warnings from Sword headers).